modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/Billing.java

```diff
@@ -6,18 +6,30 @@
 
 package org.eclipse.xpanse.modules.ocl.loader.data.models;
 
+import jdk.jfr.Description;
 import lombok.Data;
+import org.eclipse.xpanse.modules.ocl.loader.data.models.enums.BillingCurrency;
+import org.eclipse.xpanse.modules.ocl.loader.data.models.enums.BillingPeriod;
 
 /**
  * Defines the billing model of the managed service.
  */
 @Data
 public class Billing {
 
+    @Description("The business model of the managed service")
```

Comment:
All fields here must be marked as mandatory. 
String fields addtionally as "nonempty"